controller/iteration_blackbox_test.go

```diff
@@ -763,3 +765,208 @@ func assertChildIterationLinking(t *testing.T, target *app.Iteration) {
 	require.NotNil(t, target.Relationships.Parent.Links)
 	require.NotNil(t, target.Relationships.Parent.Links.Self)
 }
+
+// TestIterationDelete tests iteration delete API
+func (rest *TestIterationREST) TestIterationDelete() {
```

Comment:
for this test, you could use an extra layer of subtests, for example to distinguish valid cases vs errors (unauthorized, user not being the space owner, etc.)